src/pkg/repository/repository.go

```diff
@@ -345,6 +266,68 @@ func (r repository) Counter() *count.Bus {
 	return r.counter
 }
 
+func (r *repository) addContextInfo(ctx context.Context) context.Context {
```

Comment:
Nit: for specificity.  "info" is a little ambiguous.
```suggestion
func (r *repository) addContextClues(ctx context.Context) context.Context {
```